src/components/CommandBar/CommandBar.less

```diff
@@ -334,19 +343,14 @@
   &.ms-CommandBarSearch--hasBack .ms-CommandBarSearch-iconArrowWrapper {
     display: block;
   }
-}
-
 
-//== Modifier: After search is active and has text, add X for clearing the box
-//
-.ms-CommandBarSearch--active.ms-CommandBarSearch--hasText {
-    & .ms-CommandBarSearch-input {
-      padding-right: @SearchBox-height;
-    }
+  & .ms-CommandBarSearch-input {
```

Comment:
is .ms-CommandbarSearch-input supposed to be a child? & is usually used for adjoining classes/states--you might not need it here